packages/polaris-viz/src/components/FunnelChartNext/utilities/get-tooltip-position.ts

```diff
@@ -0,0 +1,47 @@
+import type {
+  TooltipPosition,
+  TooltipPositionParams,
+} from 'components/TooltipWrapper';
+
+import {TOOLTIP_POSITION_DEFAULT_RETURN} from '../../TooltipWrapper';
+import {eventPointNative} from '../../../utilities';
+
+interface Props {
+  tooltipPosition: TooltipPositionParams;
+  step: number;
+  maxIndex: number;
+  yMax: number;
+  formatPositionForTooltip: (index: number) => TooltipPosition;
+}
+
+export function getTooltipPosition({
```

Comment:
@envex does this need any rethinking based on recent tooltip changes?

Reply:
I don't _think_ so because this handles tooltips on it's own, not like the other charts.

Probably good to just spot check that it works fine on mobile and `chart further down/to the left and right on the page` like you mentioned above.